{"uid":"32b0180766b843c6","name":"test_ingredients_list","fullName":"tests.test_ingredients.TestIngredients#test_ingredients_list","historyId":"614551ca7970e22e71d92ff29a709595","time":{"start":1689185539879,"stop":1689185539880,"duration":1},"status":"broken","statusMessage":"requests.exceptions.ConnectionError: HTTPConnectionPool(host='localhost', port=80): Max retries exceeded with url: /api/ingredients/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb9570f1c50>: Failed to establish a new connection: [Errno 111] Connection refused'))","statusTrace":"E   ConnectionRefusedError: [Errno 111] Connection refused\n\nThe above exception was the direct cause of the following exception:\nE   urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x7fb9570f1c50>: Failed to establish a new connection: [Errno 111] Connection refused\n\nThe above exception was the direct cause of the following exception:\nE   urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=80): Max retries exceeded with url: /api/ingredients/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb9570f1c50>: Failed to establish a new connection: [Errno 111] Connection refused'))\n\nDuring handling of the above exception, another exception occurred:\nE   requests.exceptions.ConnectionError: HTTPConnectionPool(host='localhost', port=80): Max retries exceeded with url: /api/ingredients/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb9570f1c50>: Failed to establish a new connection: [Errno 111] Connection refused'))","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"story","value":"Тест получения списка рецептов."},{"name":"feature","value":"Тесты получения списка рецептов."},{"name":"tag","value":"test_ingredients_list"},{"name":"tag","value":"ingredients_tests(scope='class')"},{"name":"tag","value":"order(6)"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_ingredients"},{"name":"subSuite","value":"TestIngredients"},{"name":"host","value":"fv-az507-433"},{"name":"thread","value":"2317-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_ingredients"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["ingredients_tests(scope='class')","test_ingredients_list","order(6)"]},"source":"32b0180766b843c6.json","parameterValues":[]}